script.js

```diff
@@ -0,0 +1,82 @@
+//turn
+
+var turn = 0;
+var squares = $("td");
+
+addLetter = function() {
+
+  //if turn = even
+  if (turn % 2 === 0) {
+    //click event should append X to appropriate square
+    console.log(this);
+    $(this).text("X");
+    winner();
+  }
+  //if turn = odd
+  else {
+    //click event should append O to appropriate square
+    console.log(this);
+    $(this).text("O");
+    winner();
+  }
+  turn += 1;   //Even though this action is in the function scope, it is updating the value of a global variable.  So even thought turn = 0 in global scope, after you click, it refers to the global variable and updates it by one.
+};
+
+//For loop to assign eventlistners
+
+
+for(var i = 0; i < squares.length; i++ ){
+  squares.eq(i).on("click", addLetter);
+}
+
+//Determine winner and end game
+
+
+function winner() {
+  //row winner
+  console.log(squares.eq(0).text() + " zero");
+  console.log(squares.eq(1).text() + " one");
+  console.log(squares.eq(2).text() + " two");
+  if ((squares.eq(0).html() === squares.eq(1).html()) && (squares.eq(0).html() === squares.eq(2).html())) {
+    alert("Row 1 Winner!");
+  }
+  console.log("test");
```

Comment:
this is only specific to one row. Try and take a look at the solution to get an indicator of how to modularize this code. There's 8 specific win conditions for tic tac toe. Try to abstract that into some kind of data structure and loop through that to figure out winners.